plugin.json

```diff
@@ -59,6 +59,10 @@
                 "help_text": "How should the plugin choose a meeting name? Some names may cause conflicts if you use them on a public server, like meet.jit.si.",
                 "default": "english-titlecase",
                 "options": [
+                    {
+                        "display_name": "Ask for meeting type when you start a meeting",
```

Comment:
@jasonblais maybe you want to rephrase this option.

Reply:
```suggestion
                        "display_name": "Allow user to select meeting name",
```